README.md

```diff
@@ -57,6 +57,24 @@ Example in powershell:
    $SINCE=2020-03-01T15:00:00.000-05:00
    ```
 
+## UNTIL
+
+If you only want claims data updated or filed before a certain date specify the `$UNTIL` parameter.
+The expected format is yyyy-MM-dd'T'HH:mm:ss.SSSXXX+/-ZZ:ZZ which follows ISO datetime standards.
+
+For requests using FHIR R4, a default `_until` value is supplied if one is not provided. The value of the default `_until`
```

Comment:
Can we replace this line "For requests...parameter is set to current date and time" with: 

"This parameter is only available with V2 (FHIR R4). If no `_until` date is specified or you use a date from the future, it will default to the current date and time."